src/components/dashboard/FaceVerification/index.web.js

```diff
@@ -4,12 +4,12 @@ import { noop } from 'lodash';
 import useZoomSDK from './hooks/useZoomSDK';
 import useZoomVerification from './hooks/useZoomVerification';
 
-const FaceVerification = ({ onSuccess = noop, onError = noop }) => {
-  const completionHandler = useCallback((isSuccess, lastResult) => {
+const FaceVerification = ({ screenProps, onError = noop }) => {
+  const completionHandler = useCallback(isSuccess => {
     if (isSuccess) {
-      onSuccess(lastResult);
+      screenProps.pop({ isValid: true })
     }
-  }, [onSuccess]);
+  }, [onSuccess, screenProps]);
```

Comment:
@serdiukov-o-nordwhale onSuccess is unnecessary here after fix

Reply:
fixed during linting